metadata-ingestion/src/datahub/ingestion/source/redshift/redshift.py

```diff
@@ -641,6 +644,7 @@ def gen_schema_fields(self, columns: List[RedshiftColumn]) -> List[SchemaField]:
             if col.sort_key:
                 tags.append(TagAssociationClass(make_tag_urn(Constants.TAG_SORT_KEY)))
 
+            logger.debug(f"col.data_type = {col.data_type}")
```

Comment:
let's not log this - will be overwhelming

Reply:
done